scripts/data-prepartion/utils.py

```diff
@@ -63,21 +64,70 @@ def output_folder_path_validation(output_folder_path, classes):
 
 def get_classes_from_input_folder(input_folder_path):
     dot_files = glob.glob(input_folder_path+"/.*")
-    dot_files = [x.replace(input_folder_path+"/","") for x in dot_files]
-    classes = set(os.listdir(input_folder_path)) - set(dot_files)
+    dot_files = [x.replace(input_folder_path,"") for x in dot_files]
+    folders_list = set(os.listdir(input_folder_path)) - set(dot_files)
+    print(folders_list)
     final_classes = []
-    for _class in classes:
-        if os.path.isdir(os.path.join(input_folder_path, _class)):
-            final_classes.append(_class)
+    for _folder in folders_list:
+        if os.path.isdir(os.path.join(input_folder_path, _folder)): #Only keep folders
+            files_list = set(os.listdir(os.path.join(input_folder_path, _folder)))
+            if files_list: #Discard empty folders
+                print("Full folder: {} ".format(_folder))
+                print(list(files_list)[:10])
+                stop = False;
```

Comment:
Can you move this block into a separate function.
That is just to ensure that the code stays readable.